config/sync/search_api.index.content.yml

```diff
@@ -117,6 +119,7 @@ tracker_settings:
     indexing_order: fifo
 options:
   cron_limit: 50
+  delete_on_fail: true
```

Comment:
This is a default option, that came just from saving the index. It also exists in the other.